controllers/authController.js

```diff
@@ -1,6 +1,9 @@
 const HttpStatus = require('http-status-codes')
 const { createTkn } = require('../utils')
 const sendEmail = require('../utils/sendMail')
+const bcrypt = require('bcrypt')
+const saltRounds = 10
+const salt = bcrypt.genSaltSync(saltRounds)
```

Comment:
We don't need to generate a salt, will be autogenerated. see technique 2 from bcrypt.